Documentation/creating-new-package.md

```diff
@@ -0,0 +1,29 @@
+# Creating new .NET IoT package
+
+This is a list of steps which need to happen in order to create new package version of .NET IoT.
+
+## Steps
+
+- regenerate device listing (`dotnet run` on https://github.com/dotnet/iot/tree/master/tools/device-listing and fix all warnings)
```

Comment:
This step is most likely after checking the PR are all merged.
Also, for the stapes, you can use the pattern: 
```
- Step 1: *manual*: something
  - sub items
  - sub items
-  Step 2: *automatic*: something else
```

Reply:
+1. Also steps should begin with capital letter and end with period.